deployment/ansible/roles/cac-tripplanner.transitfeed/tasks/main.yml

```diff
@@ -1,4 +1,3 @@
 ---
 - name: Install transitfeed
-  pip: name=pytz
```

Comment:
One task with two 'pip' keys only installs the second (at least in Ansible 2.1. I guess that could have changed?), so I figured if it hasn't been actually doing anything it wasn't needed.  I'll put it back in its own task.  Do we want to peg the version?  Looks like the transitfeed project doesn't.